test_requirements/requirements_base.txt

```diff
@@ -1,19 +1,19 @@
-coverage<5
-python-coveralls==2.5.0
+coverage>=4
+python-coveralls>2.5.0
 unittest-xml-reporting==1.11.0
 Pillow==5.2.0
-django-treebeard>=4.3
+django-treebeard>=4.3,<4.5
 argparse
 dj-database-url
 djangocms-admin-style>=1.5
-django-sekizai>=0.7
-django-classy-tags>=0.7.2
-https://github.com/divio/djangocms-text-ckeditor/archive/master.zip
+django-sekizai>=0.7,<=1.10
+django-classy-tags>=0.7.2,<2
+djangocms-text-ckeditor<4
 https://github.com/ojii/django-better-test/archive/8aa2407d097fe3789b74682f0e6bd7d15d449416.zip#egg=django-better-test
 https://github.com/ojii/django-app-manage/archive/65da18ef234a4e985710c2c0ec760023695b40fe.zip#egg=django-app-manage
 iptools
 sphinx==1.8.5
-sphinxcontrib-spelling
+sphinxcontrib-spelling<7.0.0 # restriction for py35 tests
 # required to run the server for integration tests
-django-app-helper
+django-app-helper<3.0.0
```

Comment:
is this also related to python 2.x support? if yes see above

Reply:
This one is 2.2 too, as Angelo has identified.

Reply:
So we will need `django-app-helper` in the django_3.x files and `django-app-helper<3.0.0` in the django_2.2.txt file.